lms/envs/aws.py

```diff
@@ -445,6 +454,18 @@
 for database_name in DATABASES:
     DATABASES[database_name]['ATOMIC_REQUESTS'] = True
 
+# Override database settings with environment variables, if any
+for name, database in DATABASES.items():
+    if name != 'read_replica':
+        database.update({
+            'ENGINE': os.environ.get('DB_ENGINE', database['ENGINE']),
```

Comment:
Hi Omar, that makes sense, however I'd still rather keep the old environment variables for a couple of reasons.

- This pattern is used in other edx projects and I would like them all to be the same. [Example](https://github.com/edx/ecommerce/blob/master/ecommerce/settings/production.py#L50-L57)
- While the DB config can be updated for more reasons, I think it's better to show the reason that this was put in here.  By making the name more generic, we lose the intent of the code.  This is the closest thing we have to a contract here so I'd rather keep it than make the usage more ambiguous.